scripts/CRLP_TEST_VALIDATE_ROLLUPS.cls

```diff
@@ -25,8 +25,10 @@ public class CRLP_TEST_VALIDATE_ROLLUPS {
 
     // TODO -- Add support for verifying that dupe OCR's for Soft Credit Contact Roles are not duped in results
 
+    private static Boolean ROLLUPS_INCLUDES_MEMBERSHIP = true;
+
     // When testing in a multicurrency org, set up CAD with this rate: 1.2311
-    private static final Double CURR_EXCHANGE_RATE = 1.2311;
+    private static final Double CURR_EXCHANGE_RATE = 2.0;
```

Comment:
@force2b looks like you've got some unrelated changes here and below...

Reply:
I included this just because I meant to check this in earlier, but forgot. It's a script file, so I thought this was close enough. Happy to do as a separate PR too.

Reply:
Oh yes, I missed that it's in scripts. That's fine.